src/stories/components/JsPsych/JsPsych.js

```diff
@@ -0,0 +1,241 @@
+            "<div style='width: 900px; margin: auto;'>" +
+            "<div class='float: left;'><img width='300' src='" + oriFilePath + "'/>" +
+            "</div>" +
+            "<div style='float: left; width: 300px; height: 300px;'>" +
+            "<div style='font-size: 60px; width:300px height: 30px; margin-top: 135px; margin-bottom: 135px;'>+</div>" +
+            "</div>" +
+            "<div class='float: left;'><img width='300' src='" + invFilePath + "'/>" +
+            "</div>" +
+            "</div>" :
+            // For the last 200 images that are rendered, show inverted on left & show original on right
+            "<div style='width: 900px; margin: auto;'>" +
+            "<div class='float: left;'><img width='300' src='" + invFilePath + "'/>" +
+            "</div>" +
+            "<div style='float: left; width: 300px; height: 300px;'>" +
+            "<div style='font-size: 60px; width:300px height: 30px; margin-top: 135px; margin-bottom: 135px;'>+</div>" +
+            "</div>" +
+            "<div class='float: left;'><img width='300' src='" + oriFilePath + "'/>" +
+            "</div>" +
+            "</div>";
+
+        const newStimuli = {
+          stimulus: twoStimulusHtml,
+          data: {label: 'trial', trial_num: i},
+        };
+        trials.push(newStimuli);
+      }
+      return trials;
+    };
+
+    const fixation = {
+      type: 'html-keyboard-response',
+      stimulus: '<div style="height:307px; width:900px;"><div style="width:900px; height: 30px;  color:red; margin-top:135px; font-size:60px;">+</div></div>',
+      choices: jsPsych.NO_KEYS,
+      trial_duration: 1000, // From 1500
+      data: {label: 'fixation'},
+    };
+
+    const trial = {
+      type: 'html-keyboard-response',
+      stimulus: jsPsych.timelineVariable('stimulus'),
+      choices: ['e', 'i'],
+      data: jsPsych.timelineVariable('data'),
+      trial_duration: 1000, // 1000,
+      post_trial_gap: 0,
+      on_finish: function(data) {
+        console.log(data);
+        if (data.response === 'e') {
+          data.selection = 'left';
+        } else if (data.response === 'i') {
+          data.selection = 'right';
+        } else {
+          data.selection = 'none';
+        }
+      },
+    };
+
+    const postTrialPause = {
+      type: 'html-keyboard-response',
+      stimulus: '<div style="height:307px; width:900px;"><div style="width:900px; height: 30px;  color:blue; margin-top:135px; font-size:60px;">+</div></div>',
+      choices: jsPsych.NO_KEYS,
+      trial_duration: 250, // From 1500
+      data: {label: 'post-fixation'},
+    };
+
+    // Call backend api storeExperimentResult to connect with FireBase and update Users Collection with experiment data.
+    async function saveExperimentData(experimentData) {
+      const newExperimentResult = {
+        experimentId: experimentData.get().json('pretty'),
+      };
+      storeExperimentResult(participantId, experimentId, newExperimentResult);
```

Comment:
FYI: Depending on the format of the data we want, this is where we are connecting with firebase to upload experimental data. If we want to filter out & pin point certain data points only, we would modify line 181.